### DISCUSSION TITLE
Need fix for CVE-2022-41881 io.netty_netty-codec

### DISCUSSION THREAD

**Discussion by Tejeshwarraju** [tchiotludo/akhq#1367] [Q&A] [answered]
Hi Team,

Good Day !

We see a High Severity Security Vulnerability is open in tchiotludo/akhq:0.23.0, which  has a fix in  io.netty_netty-codec 4.1.86.Final version of package.

Can you provided the updated container with the fix ?

let me know if you need more details.

-Regards,
Tej

**Comment by tchiotludo** [comment, marked as answer]
I think it's already fix on dev branch, can you valid the fix and open an issue if not?

**Comment by Tejeshwarraju** [comment]
After scanning container tchiotludo/akhq:dev for  vulnerabilities  can still see the  io.netty_netty-codec needs to be upgraded to 4.1.86.Final.

Below is the snippet from scan.

`--------+
|      CVE       | SEVERITY | CVSS |          PACKAGE          |      VERSION      |        STATUS         | PUBLISHED  | DISCOVERED |                    DESCRIPTION                     |
+----------------+----------+------+---------------------------+-------------------+-----------------------+------------+------------+----------------------------------------------------+
| CVE-2022-41881 | high     | 7.50 | io.netty_netty-codec      | 4.1.84            | fixed in 4.1.86       | 55 days    | < 1 hour   | Netty project is an event-driven asynchronous      |
|                |          |      |                           |                   | 48 days ago           |            |            | network application framework. In versions prior   |
|                |          |      |                           |                   |                       |            |            | to 4.1.86.Final, a StackOverflowError can be       |
|                |          |      |                           |                   |                       |            |            | raised whe...`
